### PR TITLE
Generate various class synopses from stubs - part 2

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

**Review by Fizie92 — COMMENTED** [review]
# @

File: language/predefined/fibererror/construct.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  </para>
+
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+</refentry>
```
Comment (cmb69) [inline language/predefined/fibererror/construct.xml:24]: Maybe
```suggestion
 <refsect1 role="errors">
  &reftitle.errors;
  <para>
   Throws an <classname>Error</classname> exception when called.
  </para>
 </refsect1>
</refentry>
```

File: reference/dir/directory.xml [hunk 2]
```diff
@@ -24,30 +22,34 @@
...
       <classname>Directory</classname>
      </ooclass>
     </classsynopsisinfo>
-<!-- }}} -->
-    
+
     <classsynopsisinfo role="comment">&Properties;</classsynopsisinfo>
     <fieldsynopsis>
      <modifier>public</modifier>
+     <modifier>readonly</modifier>
```
Comment (cmb69) [inline reference/dir/directory.xml:38]: The `path` is [readonly only as of PHP 8.1.0](https://3v4l.org/cC3AN), so there should be a changelog entry. I assume it's the same with the `handle` property.

**Review by cmb69 — COMMENTED** [review]
Thank you!

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**